.gitattributes

```
@@ -0,0 +1,2 @@
+build/** linguist-generated
```

```suggestion
build/** linguist-generated
docs/** linguist-documentation
```

while we're at it... let's make also use of `linguist-documentation` to hide docs from the stats...